src/main/webapp/app/exercises/shared/plagiarism/plagiarism-header/plagiarism-header.component.ts

```diff
@@ -39,13 +40,16 @@ export class PlagiarismHeaderComponent {
      */
     denyPlagiarism() {
         if (this.comparison.status === PlagiarismStatus.CONFIRMED) {
-            this.askForConfirmation(() => this.updatePlagiarismStatus(PlagiarismStatus.DENIED));
+            this.askForConfirmationOfDenying(() => this.updatePlagiarismStatus(PlagiarismStatus.DENIED));
         } else {
             this.updatePlagiarismStatus(PlagiarismStatus.DENIED);
         }
     }
 
-    private askForConfirmation(onConfirm: () => void) {
+    private askForConfirmationOfDenying(onConfirm: () => void) {
+        // change the status to dirty so that both buttons are disabled
+        this.plagiarismStatusDirty = true;
```

Comment:
Why is a denied plagiarism case considered 'dirty'?
Or is the name of the variable misleading?

Reply:
The dirty flag is set here, so that both buttons are disabled when pop up is on

Reply:
I get that.
But this does not explain why it is considered `dirty`. If the name of the variable does not make sense in this context please change it to something like `disableConfirmDenyButton`

Reply:
Agree!
I did the change. I also added one missing flow on cancelling the pop up